src/ontology/mondo-ingest.Makefile

```diff
@@ -336,15 +336,15 @@ build-mondo-ingest:
 		mapped-deprecated-terms mapping-progress-report \
 		recreate-unmapped-components sync documentation \
 		prepare_release
-	echo "Mondo Ingest has been fully completed"
+	@echo "Mondo Ingest has been fully completed"
```

Comment:
## ODK PR
Note also that the same issue also exists in the `Makefile`, e.g. `echo Finished running all tests successfully.`, which comes from ODK. So I'd like to do the same quick PR in ODK as well.